### ISSUE TITLE
Bug: Value error due to underscore in `arbitrum_one`

### ISSUE THREAD

**Issue by rapcmia** [hummingbot/gateway#173]
**Describe the bug**
When using arbitrum for DEX connector on scripts like `amm_data_feed_example`, we are getting a returning `ValueError: too many values to unpack (expected 3)` 

![image](https://github.com/hummingbot/gateway/assets/73840223/c673162d-f1f0-422e-a0d8-ade32ed76a13)

```
2023-07-18 14:15:33,851 - 8163 - GatewayEVMAMM - INFO - Network status has changed to NetworkStatus.CONNECTED. Starting networking...
2023-07-18 14:15:34,001 - 8163 - hummingbot.strategy.script_strategy_base - WARNING - uniswap_ethereum_arbitrum_one is not ready. Please wait...
2023-07-18 14:15:35,000 - 8163 - hummingbot.strategy.script_strategy_base - WARNING - uniswap_ethereum_arbitrum_one is not ready. Please wait...
2023-07-18 14:15:36,001 - 8163 - hummingbot.strategy.script_strategy_base - WARNING - uniswap_ethereum_arbitrum_one is not ready. Please wait...
2023-07-18 14:15:37,001 - 8163 - hummingbot.strategy.script_strategy_base - WARNING - uniswap_ethereum_arbitrum_one is not ready. Please wait...
2023-07-18 14:15:39,002 - 8163 - hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "/Users/rapcomia/github/hummingbot/development/hummingbot/core/utils/async_utils.py", line 9, in safe_wrapper
    return await c
  File "/Users/rapcomia/github/hummingbot/development/scripts/amm_price_example.py", line 31, in async_task
    connector, chain, network = self.connector_chain_network.split("_")
ValueError: too many values to unpack (expected 3)
2023-07-18 14:15:39,965 - 8163 - hummingbot.core.rate_oracle.rate_oracle - INFO - Network status has changed to NetworkStatus.CONNECTED. Starting networking...
```

**Steps To Reproduce**
Use `uniswap_ethereum_arbitrum_one` when running AMM scripts found on `./scripts` folder

**Release version**
dev-1.19.0

**Comment by Arnique** [comment]
This is an old bug that I've never seen raised. The current native code separates the connector, chain, and network by splitting the connector string using the `"_"` character. The presence of 4 `"_"` instead of the expected 3 in the string `"uniswap_ethereum_arbitrum_one"` leads to the malfunction.

 A simple fix to your script is to set the connector, chain, and network directly instead of using one string i.e
 
```
    self.amm_data_feed_uniswap.connector = "uniswap"
    self.amm_data_feed_uniswap.chain = "ethereum"
    self.amm_data_feed_uniswap.network = "arbitrum_one"
```

**Comment by fengtality** [comment]
## Bounty

* Description: Change all references to `arbitrum_one` chain to `arbitrum`
* Acceptance criteria: Change all files, including tests, etc
* Bounty: P3
* Bounty Amount: 10,000
* Status: OPEN

**Comment by isreallee82** [comment]
that's not the case @Arnique 
is this still open I will like to work on it @fengtality

**Comment by isreallee82** [comment]
I noticed there are possible scenarios, where connector chain and network are can be used in different arrangements.

will still like to work on this

**Comment by nikspz** [comment]
This issue's assigned to @isreallee82

**Comment by nikspz** [comment]
https://github.com/hummingbot/gateway/pull/182

**Comment by nikspz** [comment]
@isreallee82  please fill up & submit the AML Policy Form via this link: https://forms.gle/ZeSRvWUU3gBeHPbH6
This is a one-time process for each ETH Address used to receive HBOT bounty.

**Comment by isreallee82** [comment]
done

**Comment by nikspz** [comment]
Closing this bounty as paid, fix provided in 1.19.0 release
